docs/_posts/2015-12-09-ismounted-antipattern.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "isMounted is an Antipattern"
+author: jimfb
+---
+
+The primary use case for `isMounted()` is to avoid calling `setState()` after a component has unmounted, because calling `setState()` after a component has unmounted will emit a warning. The “setState warning” exists to help you catch bugs, because calling `setState()` on an unmounted component is an indication that your app/component has somehow failed to clean up properly. Specifically, calling `setState()` in an unmounted component means that your app is still holding a reference to the component after the component has been unmounted - which often indicates a memory leak!
+
+To avoid the error message, people often add lines like this:
+
+```js
+if(this.isMounted()) this.setState({...}); // This is bad.
+```
+
+Checking `isMounted` before calling `setState()` does eliminate the warning, but it also defeats the purpose of the warning, since now you will never get the warning (even when you should!)
+
+Most other uses of `isMounted()` are similarly erroneous; using `isMounted()` is usually a code smell.
+
+An easy migration strategy for anyone upgrading their code to avoid `isMounted()` is to track the mounted status yourself.  Just set a `_isMounted` property to true in `componentDidMount` and set it to false in `componentWillUnmount`, and use this variable to check your component's status.
```

Comment:
For instance, because you want to upgrade to ES6 classes, which don't support the function.  Or because you want to start preparing your codebase for when we do delete it.